modules/shared/partials/acid-transactions.adoc

```diff
@@ -41,7 +34,7 @@ There are two higher durability settings available that will additionally wait f
 This further increases safety, at a cost of additional latency.
 
 A level of `None` is present but its use is discouraged and unsupported.
-If durability is set to `None`, then atomicity _cannot be guaranteed_.
+If durability is set to `None`, then ACID semantics are not guaranteed.
```

Comment:
you should be able to replace the word "meanings" anywhere you see semantics and still have the sentence make sense.  It feels a bit to me like this goes overboard with this phrase.  Not a strong opinion, just food for thought.

Reply:
Fair enough, I'll fix the semantics (used it correctly that time I think).